user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -773,4 +783,35 @@ public static native void onTabSelect( Element element )/*-{
         // ignore
     }
   }-*/;
+
+  public void setIsAsyncModeEnabled() {
```

Comment:
Please add a explaining comment here so that other developers know what this is and why it is there. The "mantle" code is shared and developers who have not worked in the report-viewer may not know anything about the work we do.

A short explanation like: "Queries the server once to see whether report-viewer async-mode is enabled. If it is enabled, the report-viewer provides it own loading screen and the default "please wait" screen interferes with the user experience of that. Therefore we disable the wait-screen that comes up when a new page opens. @see case BACKLOG-xxxx for details."

This way anyone reading the code in a year or two will know why this is here.